packages/gene-tools/src/generators/service-generator/index.spec.ts

```diff
@@ -140,21 +140,22 @@ describe('Service generator', () => {
       .read('libs/question/services/question-service/src/index.ts')
       ?.toString();
 
-    expect(indexFile).toContain(
-      "export { useQuestions, queryQuestions, getQuestionsQueryKey } from './lib/useQuestions';"
-    );
-    expect(indexFile).toContain(
-      "export { useQuestion, queryQuestion, getQuestionQueryKey } from './lib/useQuestion';"
-    );
-    expect(indexFile).toContain(
-      "export { useCreateQuestion } from './lib/useCreateQuestion';"
-    );
-    expect(indexFile).toContain(
-      "export { useDeleteQuestion } from './lib/useDeleteQuestion';"
-    );
-    expect(indexFile).toContain(
-      "export { useUpdateQuestion } from './lib/useUpdateQuestion';"
-    );
+    expect(indexFile).toMatchInlineSnapshot(`
```

Comment:
looks like those nothing more that snapshost tests, so updated. This way it's easier to manage those in case of formatting change